pinakes/main/common/views.py

```diff
@@ -26,6 +26,13 @@
     ),
     list=extend_schema(
         description="List all groups",
+        parameters=[
```

Comment:
@bzwei These are optional parameters correct?

Reply:
yes. not required